src/pages/home/ui/HomePageSection/index.tsx

```
@@ -0,0 +1,222 @@
+import MainHamImg2 from '@/shared/assets/pngs/MainHamImg2.png';
+import Text from '@/shared/ui/Text';
+import { ArrowRight, Bell, Menu, Next } from '@/shared/assets/svgs/icon';
+import { useNavigate } from 'react-router';
+import { ROUTE } from '@/shared/config/route';
+import { useState } from 'react';
+import CoinImg from '@/shared/assets/pngs/CoinImg.png';
+import LinkMain from '@/shared/assets/pngs/link_main.png';
+import CardMain from '@/shared/assets/pngs/card_main.png';
+
+import Flex from '@/shared/ui/Flex';
+import HomeExpenseItem from '../HomeExpenseItem';
```

ESLint

[error] `../HomeExpenseItem` import should occur after import of `@/shared/ui/Chip`

(import/order)